stan/math/opencl/kernel_generator/load.hpp

```diff
@@ -49,7 +49,7 @@ class load_
    * Creates a deep copy of this expression.
    * @return copy of \c *this
    */
-  inline load_<T&> deep_copy() const& { return load_<T&>(a_); }
+  inline load_<T&> deep_copy() const & { return load_<T&>(a_); }
```

Comment:
Can you revert this change? Without this change the tests will pass without running the entire suite.

Reply:
Maybe... I haven't asked for this change - the Jenkins did this for me. Let me try.

Reply:
Ok, I see why this happened (some machine has clang-format 5.0.0 and some have 6.0.0). Dont bother too much if it wont work.